runtime/karura/src/lib.rs

```diff
@@ -2218,7 +2236,7 @@ mod tests {
 	#[test]
 	fn check_call_size() {
 		assert!(
-			core::mem::size_of::<Call>() <= 260,
+			core::mem::size_of::<Call>() <= 300,
```

Comment:
why increased?

Reply:
This is commited by mistake. I'll revert this